Sources/App/migrations.swift

```diff
@@ -0,0 +1,48 @@
+import Vapor
+import Fluent
+import FluentPostgresDriver
+
+public func migrations(_ app: Application) throws {
```

Comment:
Should we namespace this somehow, so its not just available everywhere? Like as an extension on `App` or even just `Migrations.migrate(app)`

Entirely for sake of autocomplete convenience, not a big deal!

Reply:
I had copied the free floating function idea / approach from configure & routes as Vapor sets it up, but yeah might be a bit dangerous to have people calling them anywhere / showing up in auto-complete as global functions. Will pop it behind Migrations class as a static function so it's at least scoped.

Reply:
Updated to be a static function inside Migrations class.

Reply:
Ah gotcha! Yeah their kinda "kick-off" functions feel like something that should be fairly one off like the main function. 

Definitely something entirely optional, the name space feels like a clean approach without going _too_ far, but maybe more experienced Vapor folks would prefer how you had it - hopefully this works for us going forwards